brave-core/src/main/java/com/github/kristofa/brave/HttpSpanCollector.java

```diff
@@ -0,0 +1,198 @@
+package com.github.kristofa.brave;
+
+import com.github.kristofa.brave.internal.Nullable;
+import com.twitter.zipkin.gen.Span;
+import java.io.Closeable;
+import java.io.Flushable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayDeque;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.Executors;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import okio.Buffer;
```

Comment:
these are re-packaged and minified. adds 50k I think to the jar file. I think it is worth it, as brave is still under 256k including other imports we've done.